<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientOrServer</class>
 <widget class="QMainWindow" name="ClientOrServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#serverButton{ 
background-color: qlineargradient(spread:pad, x1:0.0432692, y1:0.051, x2:0.952, y2:0.892045, stop:0.423077 rgba(152, 118, 63, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(8, 8, 8);
font: 900 16pt &quot;Bodoni MT Black&quot;;
}
QPushButton#serverButton:hover{ 
background-color: rgb(156, 119, 56);
}
QPushButton#serverButton:pressed{ 
padding-left:5px;
padding-top:5px;
background-color: rgb(200, 158, 73);
}

QPushButton#clientButton{ 
background-color: qlineargradient(spread:pad, x1:0.0432692, y1:0.051, x2:0.952, y2:0.892045, stop:0.423077 rgba(152, 118, 63, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(8, 8, 8);
font: 900 16pt &quot;Bodoni MT Black&quot;;
}
QPushButton#clientButton:hover{ 
	background-color: rgb(156, 119, 56);
}
QPushButton#clientButton:pressed{ 
padding-left:5px;
padding-top:5px;
background-color: rgb(200, 158, 73);
}
</string>
    </property>
    <widget class="QLabel" name="basePicture">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Prefix/resource/download (2).jpg);
border:3px solid; 
border-color :#000</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="clientButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#clientButton{ 

background-color: qlineargradient(spread:pad, x1:0.0432692, y1:0.051, x2:0.952, y2:0.892045, stop:0.423077 rgba(152, 118, 63, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(8, 8, 8);
border-radius:15px;
font: 900 16pt &quot;Bodoni MT Black&quot;;
}
QPushButton#clientButton:hover{ 
	background-color: rgb(156, 119, 56);
border-radius:15px;
}
QPushButton#clientButton:pressed{ 
padding-left:5px;
padding-top:5px;
	background-color: rgb(200, 158, 73);
}
</string>
     </property>
     <property name="text">
      <string>Client</string>
     </property>
    </widget>
    <widget class="QPushButton" name="serverButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>200</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton#serverButton{ 

background-color: qlineargradient(spread:pad, x1:0.0432692, y1:0.051, x2:0.952, y2:0.892045, stop:0.423077 rgba(152, 118, 63, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(8, 8, 8);
border-radius:15px;
font: 900 16pt &quot;Bodoni MT Black&quot;;
}
QPushButton#serverButton:hover{ 
	background-color: rgb(156, 119, 56);
border-radius:15px;
}
QPushButton#serverButton:pressed{ 
padding-left:5px;
padding-top:5px;
	background-color: rgb(200, 158, 73);
}

</string>
     </property>
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
    <widget class="QLabel" name="ipLable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>591</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>491</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
color: rgb(0, 0, 0);
border:none;
font: 16pt &quot;Snap ITC&quot;;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>300</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>basePicture</zorder>
    <zorder>ipLable</zorder>
    <zorder>textBrowser</zorder>
    <zorder>label</zorder>
    <zorder>serverButton</zorder>
    <zorder>clientButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
